src/SUFST/src/can_unpack.c

```diff
@@ -77,38 +113,31 @@ void queue_receive_thread_entry(ULONG input)
               break;
             }
             /* Couldn't find matching identifier - deassign pointer. */
-            if(i == TABLE_SIZE)handlerunpack = NULL;
```

Comment:
Single line `if` statements with no brackets are something MISRA C also has guidelines about. In general, every `if` statement and similar blocks should always have the braces `{}` even if they are just one line.